tests/phpunit/tests/theme/wpThemeJsonResolver.php

```diff
@@ -634,6 +634,8 @@ function test_merges_child_theme_json_into_parent_theme_json() {
 	 * @covers WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles
 	 */
 	function test_get_user_data_from_wp_global_styles_does_not_use_uncached_queries() {
+		// Switch to a theme that does have support.
+		switch_theme( 'block-theme' );
```

Comment:
Fixing tests always makes me nervous, why is this needed now? Maybe expand the comment so others can understand too. 

:1234: This applies to the other tests with the same change.

Reply:
Previously the logic here simply ignored the fact that classic themes don't need to run this logic, therefore the tests didn't bother switching to a block theme. Fixing this is precisely part of the goals of this PR, therefore the other tests that were operating under the assumption of a block theme now need to be updated to actually enforce a block theme.

So I'd argue it was rather the existing tests here that were previously too vague.